### PR TITLE
Feature/clean dependencies

### PR DESCRIPTION
**Pull request by JorxSS**
What's new:

* Fixed again Playground
* Cleaning Environment VFX dependencies.
* Fixed water zone damage again, thanks @Dartemiss.

### CODE REVIEW

File: Scripts/src/Script/EventManager.cpp [hunk 1]
```diff
@@ -447,7 +405,11 @@ void EventManager::RestartEvents()
 	{
 		battle->RestartEvent();
 	}
-
+	for (auto & root : root_obstacles)
```
Comment (Dartemiss) [inline Scripts/src/Script/EventManager.cpp:408]: CS

**Review by Dartemiss — CHANGES_REQUESTED** [review]
GJ! Works more or less everything but! It crashes when finishing the game on level1 entering the cave (see image below)

EventManager
imagen

Also you already know but particles takes too much time to spawn.

File: Scripts/src/Script/EventManager.cpp (hunk 1 above)
Comment (JorxSS) [reply Scripts/src/Script/EventManager.cpp]: Done

**Comment by JorxSS** [comment]
@Dartemiss already fixed the feature/bug now cannot happen sorry mi pana

**Review by Dartemiss — APPROVED** [review]

**Comment by Pacasasgar** [comment]
I found this bug while testing this version:
water_death_bug

As you mention in the description, water zone has been fixed. Is this another unkown bug? Player do not die but can not move either in the gif's state.

**Comment by JorxSS** [comment]
> I found this bug while testing this version:
> water_death_bug
> 
> As you mention in the description, water zone has been fixed. Is this another unkown bug? Player do not die but can not move either in the gif's state.

known, will be fixed with the new animations

**Comment by Pacasasgar** [comment]
> > I found this bug while testing this version:
> > water_death_bug
> > As you mention in the description, water zone has been fixed. Is this another unkown bug? Player do not die but can not move either in the gif's state.
> 
> known, will be fixed with the new animations

Great then!

**Review by Pacasasgar — APPROVED** [review]
Nice job Jordi!

**Pull request merged**